### DISCUSSION TITLE
Edit configuration file from UI/Tray icon

### DISCUSSION THREAD

**Discussion by Rhuax** [espanso/espanso#1448] [Feature requests and Ideas]
Hi, I think it would be super efficient to be able to access the configuration file (base.yml), from the right click on the espanso tray icon.
Everytime I have to find on google espanso path and change the file. I would be easier if I could right click on espanso tray icon, have a new menu item like "open configuration file", click it and open it with the default text editor.

**Comment by kpym** [comment]
Already requested in #1234, #1341 and #1440.
